geode-core/src/main/java/org/apache/geode/examples/security/ExampleSecurityManager.java

```diff
@@ -257,15 +257,65 @@ private void readUsers(final Map<String, User> rolesToUsers, final JsonNode node
     return roleMap;
   }
 
-  static class Role {
+  public static class Role {
     List<ResourcePermission> permissions = new ArrayList<>();
+
```

Comment:
I didn't add them, I just marked them public so that the Test could reach them.  It didn't make sense to me for TestSecurityManagerTest to be testing TestSecurityManager.  It makes slightly more sense to test the ExampleSecurityManager.  The two security managers are identical code at this point (RE: GEODE-2092), but I'm sure that'll change in the near future as we build out Geode examples.

If we were to remove Roles and Users, we'd also remove about 95% of the code in ExampleSecurityManager, not leaving much example left.  We should have a separate ticket to refactor the example.  This one really is only supposed to address some Javadoc and System.out messages.